Modix.Services/ErrorHandling/AuthResultFormatter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Discord;
+using Discord.Commands;
+using Modix.Services.Core;
+
+namespace Modix.Services.ErrorHandling
+{
+    public class AuthResultFormatter : DiscordResultFormatter<AuthResult>
```

Comment:
I think this belongs in `Modix.Bot`. It's the only frontend that needs to render embeds, specifically.